src/dspbase.jl

```diff
@@ -118,75 +118,319 @@ function deconv(b::StridedVector{T}, a::StridedVector{T}) where T
 end
 
 # padded must start at index 1, but u can have arbitrary offset
-function _zeropad!(padded::AbstractVector, u::AbstractVector)
-    datasize = length(u)
+@inline function _zeropad!(
+    padded::AbstractVector,
+    u::AbstractVector,
+    data_dest = (1,),
+    data_region = CartesianIndices(u)
+)
```

Comment:
Just for function parameters or everywhere?

Reply:
When you press `C-RET` in Juno it executes the current block of code, and determines what a "block" is by indentation, so anything one would want to evaluate as a block needs to be indented. The function parameter lists are the big one because often when testing things I'll tweak the code and then re-evaluate the function to update it.